File: week-09/day-2/groot/src/main/java/fabian/zsofia/groot/models/Groot.java
package fabian.zsofia.groot.models;

public class Groot {

    private String received;
    private String translated;

    public Groot(String received) {
        this.received = received;
        this.translated = "I am Groot";
    }

    public String getReceived() {
        return received;
    }

    public String getTranslated() {
        return translated;
    }
}

File: week-12-project-phase/day-3/IntegrationTestBasics/Models/Groot.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace IntegrationTestBasics.Models
{
    public class Groot
    {
        public string Received { get; set; }
        public string Translated { get; set; }

        public Groot(string received)
        {
            Received = received;
            Translated = "I am Groot!";
        }
    }
}
